packages/web-components/src/components/ic-card/ic-card.tsx

```diff
@@ -284,6 +286,7 @@ export class Card {
                   ["toggle-button-expanded"]: this.areaExpanded,
                 }}
                 variant="icon"
+                size="large"
```

Comment:
This makes the button 40x40 but designs are 32x32.

Reply:
I spoke to the design team on this and it was agreed that we use the large variant of the icon button so the height of the buttons (chevron & slotted buttons) are the same. Design plans to update size variants so that the default size will be 40px across all components.